lib/src/main/java/com/orange/ods/compose/component/card/OdsHorizontalCard.kt

```diff
@@ -50,47 +42,33 @@ import com.orange.ods.compose.theme.OdsTheme
  * Cards contain content and actions about a single subject.
  *
  * @param title The title to be displayed in the card.
- * @param image The painter of the card image.
+ * @param image The card image.
  * @param modifier Modifier to be applied to the layout of the card.
  * @param subtitle Optional subtitle to be displayed in the card.
  * @param text Optional text description to be displayed in the card. It is truncated to fit on 2 lines.
- * @param button1Text Optional text of the first button in the card. If not present, button will not be shown. If present, [onButton1Click] need to be handled.
- * @param button2Text Optional text of the second button in the card. If not present, button will not be shown. If present, [onButton2Click] need to be handled.
- * @param imageContentDescription Optional card image content description.
- * @param imageBackgroundColor Optional background color of the card image.
- * @param imageContentScale Optional scale parameter used to determine the aspect ratio scaling to be used
- * if the bounds are a different size from the intrinsic size of the [Painter]
- * @param imageAlignment Optional alignment parameter used to place the [Painter] in the given
- * bounds defined by the width and height.
+ * @param firstButton Optional first button in the card.
+ * @param secondButton Optional second button in the card.
  * @param imagePosition Position of the image, it can be [OdsHorizontalCardImagePosition.Start] or [OdsHorizontalCardImagePosition.End] in the card. [OdsHorizontalCardImagePosition.Start] by default.
- * @param dividerEnabled If true, a divider is displayed between card content and the action buttons. True by default.
- * @param onCardClick Optional click on the card itself.
- * @param onButton1Click Optional handler for the first button click.
- * @param onButton2Click Optional handler for the second button click.
+ * @param divider If true, a divider is displayed between card content and the action buttons. True by default.
+ * @param onClick Optional click on the card itself.
```

Comment:
Is this a convention to add `Optional` in parameter documentation? We can see it's optional in the signature. 
It's a question I've asked myself before and it's to be homogeneous everywhere.

Reply:
I took a look at Compose documentation and it does not seem to be a convention.

More generally, I think we should create an issue to update the documentation with a todo list (which would obviously contain your remark).